netbox/templates/dcim/rack_elevation_list.html

```diff
@@ -14,7 +14,7 @@
                 {% for rack in page %}
                     <div style="display: inline-block; width: 266px">
                         <div class="rack_header">
-                            <strong><a href="{% url 'dcim:rack' pk=rack.pk %}">{{ rack.name|truncatechars:"25" }}</a></strong>
```

Comment:
As with devices, we intentionally reference `name` here to avoid calling `display_name()`.